astroplan/tests/test_core.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord)
+import astropy.units as u
+from astropy.time import Time
+from astropy.tests.helper import remote_data
```

Comment:
@eteq: I'm borrowing astropy's test helpers in my tests, since `SkyCoord.from_name` requires internet access, which gets used in `FixedTarget.from_name`. Is it appropriate to steal this astropy machinery like so?